### PR TITLE
LPS-182031 [CP 2.0] Key generation form: same dates used for subscription and activation key dates

### PR DESCRIPTION
**Pull request by pedroliveirav**
[LPS-182031](https://issues.liferay.com/browse/LPS-182031)

### CODE REVIEW

**Comment by pedroliveirav** [comment]
ci:test:sf

**Comment by pedroliveirav** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/SelectSubscription.jsx [hunk 1]
```diff
@@ -152,6 +152,20 @@ const SelectSubscription = ({
...
+				subscriptionTerm.licenseKeyEndDate,
+				FORMAT_DATE_TYPES.day2DMonthSYearN
+			);
+		}
+
+		return getDateCustomFormat(
+			subscriptionTerm.endDate,
+			FORMAT_DATE_TYPES.day2DMonthSYearN
+		);
+	};
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/SelectSubscription.jsx:167]: Simplify

```suggestion
	const subscriptionTermEndDateByKeyType = (subscriptionTerm) =>
	 getDateCustomFormat(
		hasNotPermanentLicence  ? subscriptionTerm.licenseKeyEndDate : subscriptionTerm.endDate,
		FORMAT_DATE_TYPES.day2DMonthSYearN
	)
```

**Review by kevenleone — APPROVED** [review]
Thank you @pedroliveirav 

Good job on that, since the relevant failed could you apply this change? Thanksssss

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/SelectSubscription.jsx (hunk 1 above)
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/SelectSubscription.jsx]: Done. Thanks!

**Comment by pedroliveirav** [comment]
ci:test:relevant

**Review by samuelleolopes — APPROVED** [review]
Good Job! :fire:

**Comment by kevenleone** [comment]
ci:forward:force

**Comment by kevenleone** [comment]
ci:forward:force

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
resent: https://github.com/brianchandotcom/liferay-portal/pull/135156

**Pull request closed without merging**